core/api/src/services/bria/index.ts

```diff
@@ -107,7 +107,10 @@ export const BriaSubscriber = () => {
   }
 }
 
-const queueNameForSpeed = (speed: PayoutSpeed): string => briaConfig.queueNames[speed]
+const queueNameForSpeed = (speed: PayoutSpeed): string => {
+  const queue = briaConfig.payoutQueues.find((queue) => queue.speed === speed)
+  return queue!.queueName
```

Comment:
non null assertion must not be used, if it is necessary return the error properly (this can cause an exception in prod)